setup/ci/wercker.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: single
+title:  "Wercker"
+sidebar:
+  nav: setup
+---
+
+{% include toc %}
+
+[Wercker](http://www.wercker.com) is a cloud CI system, which can be 
+enabled within Spinnaker as a pipeline trigger and also as a pipeline stage. The setup required
+is described below.
+
+## Prerequisites
+
+To enable Wercker integration in Spinnaker, you will need to have:
+- A login to Wercker, which can be set up [here](https://app.wercker.com/).
+- A Wercker "personal token" to provide to Spinnaker so that it can access 
+the Wercker API on your behalf. Personal tokens can be generated on Wercker by logging in 
+and visiting your "Settings" page.
+
+## Add a Wercker 'Master' to Spinnaker Using Halyard
+A "master" is a connection to Wercker from Spinnaker. It consists of the Wercker URL 
+and credentials.
+1. First, make sure that Wercker is enabled:
+
+   ```bash
+   hal config ci wercker enable
+   ```
+
+2. Next, add a Wercker master i.e. a connection to Wercker from Spinnaker.
+      ```bash
+      hal config ci wercker master add mywercker1
+          --address https://app.wercker.com/ 
+          --user myuserid 
+          --token
+      ```
+
+3. Apply your changes:
+
+   `hal deploy apply`
+
+## Wercker as Pipeline Trigger
+When configuring a Spinnaker Pipeline, Wercker will be available as one of the automated 
+trigger type options. You will be able to select a Wercker master that you configured earlier,
```

Comment:
nit: change the description here and below to present tense ("will be available" -> "is available", etc).

Also consider linking to the [adding a trigger](https://www.spinnaker.io/guides/user/pipeline/managing-pipelines/#add-a-trigger) and [adding a stage](https://www.spinnaker.io/guides/user/pipeline/managing-pipelines/#add-a-stage) documentation when referencing those actions.

Reply:
done

Reply:
Question - is my guess correct that an anchor is generated for every heading? I'd like to link to the specific section on Wercker in the pipeline stage reference, instead of to the top of the page. Can I assume there will be a /reference/pipeline/stages#wercker to link to?

Reply:
Yep, that'd work. The links that are generated automatically, like for page TOCs, have a slightly different syntax:
reference/pipeline/stages/#wercker

And that's how we've been doing the links we create manually. 

But I just tested it without the slash before the anchor, and that works too. Prefer the former syntax, for consistency.

Reply:
Thanks - I used the anchor link with the format you suggested.